core/src/main/java/org/apache/calcite/plan/RelOptUtil.java

```diff
@@ -286,6 +286,22 @@ public static Set<CorrelationId> getVariablesUsed(RelNode rel) {
     return visitor.vuv.variables;
   }
 
+  /**
+   * Returns a set of variables used by the given list of sub-queries and its descendants.
+   *
+   * <p> Internally it uses the {@link org.apache.calcite.plan.RelOptUtil#getVariablesUsed}
+   * to get all the variables used in the relational expression of a given sub-query.
+   *
+   * <p> The item type is same as {@link org.apache.calcite.rex.RexCorrelVariable#id}.
```

Comment:
I really don't know what item you are referring to here.

Reply:
I used the same comment format as that of getVariablesUsed(RelNode rel) as both the functions return same output, In this context item type is a type parameter for Set<>

Reply:
Javadoc can document type parameters, use that format: https://www.baeldung.com/java-javadoc-generic-type-parameters